kinesis_video_streamer/launch/kinesis_video_streamer.launch.py

```diff
@@ -21,19 +21,17 @@
 # Argument names
 NODE_NAME = "node_name"
 CONFIG = "config"
+AWS_REGION = "aws_region"
+STREAM_NAME = "stream_name"
+REKOGNITION_DATA_STREAM = "rekognition_data_stream"
+
 MAX_STRING_LEN = 128
 
-def get_logger_config_path():
+def get_logger_config_path(config_yaml):
   logger_config_path = os.path.join(get_package_share_directory('kinesis_video_streamer'),
     'config', 'kvs_log_configuration')
-  default_config = os.path.join(get_package_share_directory('kinesis_video_streamer'),
-    'config', 'sample_config.yaml')
-
-  with open(default_config, 'r') as f:
-      config_text = f.read()
-  config_yaml = yaml.safe_load(config_text)
-
   fallback_path = '/tmp/kvs_log_configuration'
+
   if fallback_path == config_yaml['kinesis_video_streamer']['ros__parameters']['kinesis_video']['log4cplus_config']:
```

Comment:
These else's can all be removed if the order of checks is changed. 

```
if fallback_path != config_yaml['kinesis_video_streamer']['ros__parameters']['kinesis_video']['log4cplus_config']:
 return None
if len(logger_config_path) <= MAX_STRING_LEN:
  return logger_config_path
shutil.copy(logger_config_path, fallback_path)
return fallback_path
```
With comments etc. (They don't show in github code blocks strangely)

Reply:
Done.